src/Parser/BalanceDruid/Modules/Spells/Sunfire.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { formatPercentage } from 'common/format';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+
+class Sunfire extends Module {
+  suggestions(when) {
+    const sunfireUptimePercentage = this.owner.modules.enemies.getBuffUptime(SPELLS.SUNFIRE.id) / this.owner.fightDuration;
+
+    when(sunfireUptimePercentage).isLessThan(0.98)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span> Your <SpellLink id={SPELLS.SUNFIRE.id} /> uptime was {formatPercentage(sunfireUptimePercentage)}%, unless you have extended periods of downtime it should be near 100%.</span>)
+          .icon(SPELLS.SUNFIRE.icon)
+          .actual(`${formatPercentage(sunfireUptimePercentage)}% uptime`)
```

Comment:
Please use the `actual` variable to avoid having to keep this in sync with the `when()` argument.